### PR TITLE
[stable/cerebro] cerebro: iterate over .Values.env to set specific auth settings

### PR DESCRIPTION
**Pull request by sebdoido**
Currently, some auth settings from application.conf should be overwritter through env variables.
We can only set specific values from secret right now.

This PR allows to set some settings through env variables, and not only from secrets.

### CODE REVIEW

File: stable/cerebro/README.md [hunk 1]
```diff
@@ -62,6 +62,7 @@ The following table lists the configurable parameters of the cerebro chart and t
 | `nodeSelector`                      | Settings for nodeselector           | `{}`                                      |
 | `tolerations`                       | Settings for toleration             | `{}`                                      |
 | `affinity`                          | Settings for affinity               | `{}`                                      |
+| `env`                               | List with env vars (key/value   )   |                                           |
```
Comment (davidkarlsen) [inline stable/cerebro/README.md]:
```suggestion
| `env`                               | Map of env vars (key/value   )   | `{}`                                          |
```

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

File: stable/cerebro/README.md (hunk 1 above)
Comment (sebdoido) [reply stable/cerebro/README.md]: thanks for your review. I update the PR

**Review by davidkarlsen — APPROVED** [review]

**Pull request merged**